coordinator/src/services/translation/cancellation/cancellation-response.ts

```diff
@@ -76,39 +100,40 @@ function createBundleEntries(cancellationResponse: CancellationResponse) {
     cancelRequestId
   )
 
-  const bundleResources = [
-    fhirMessageHeader,
-    fhirMedicationRequest,
-    fhirPatient,
-    fhirAuthorPractitioner,
-    ...fhirAuthorLocations,
-    fhirAuthorHealthcareService,
-    fhirAuthorPractitionerRole,
-    fhirResponsiblePartyPractitioner,
-    ...fhirResponsiblePartyLocations,
-    fhirResponsiblePartyHealthcareService,
-    fhirResponsiblePartyPractitionerRole
-  ]
+  bundleResources.push(fhirMedicationRequest, fhirMessageHeader)
```

Comment:
I think the message header needs to be the first resource in the bundle: https://www.hl7.org/fhir/messageheader.html "The reference(s) that are the subject of the action as well as other information related to the action are typically transmitted in a bundle in which the MessageHeader resource instance is the first resource in the bundle."